src/core/main.cpp

```diff
@@ -361,7 +361,7 @@ int main( int argc, char * * argv )
 			new QCoreApplication( argc, argv ) :
 					new gui::MainApplication(argc, argv);
 
-	AudioEngine::qualitySettings qs( AudioEngine::qualitySettings::Mode::HighQuality );
+	AudioEngine::qualitySettings qs(AudioEngine::qualitySettings::Mode::Draft);
```

Comment:
This affects quality of command-line exports. Is this really desired?

Reply:
> This affects quality of command-line exports.

These quality settings do not seem to actually do anything meaningful at the moment. I set it to `Draft` since that's actually what we use by default in the main app, and 2, I knew these settings didn't really do much anyways.

Nothing is hooked up to the `AudioEngine::qualitySettingsChanged` signal, and during export, the only thing that seems to set the quality would be `AudioEngine::setAudioDevice` (which does nothing for the quality but assign to the `m_qualitySettings` object). To set the quality back after finishing an export, `RenderManager` calls `AudioEngine::changeQuality`, which only affects the resampling state for the audio device (which 1, is not used unless the engine sample rate and device sample rate differ somehow, and the only way they can differ is through oversampling since the engine gets its sample rate from the device, and 2, leaves all the other resampling states unaffected, like in the AFP). 

The design for audio quality here is confusing, since we supposedly have something here that tries to facilitate managing the quality, but have plugins like the AFP specifying their own quality settings (i.e., through its own interpolation modes), while also having no observable behavior for what we have over here. My point is, even if these settings were used, the effect the user would hear in the quality of the exported track would be lackluster.

I just want to get rid of this high-quality mode cruft because I was in the middle of refactoring the audio quality interface, and this ended up getting in the way (I didn't want too crowd my work with this in the same PR).

Reply:
Though, if you prefer, I can add the `HighQuality` option back into `AudioEngine::qualitySettings`. I have plans to refactor this whole thing later.